x-pack/legacy/plugins/siem/public/pages/case/components/user_list/index.tsx

```diff
@@ -63,6 +66,7 @@ const renderUsers = (
           onClick={handleSendEmail.bind(null, email)}
           iconType="email"
           aria-label="email"
+          isDisabled={!email}
```

Comment:
nit -> email != null like that we know we are not dealing with boolean

Reply:
thats the opposite of what i want, but happy to change to `email == null` 😄

Reply:
What about empty emails?

Reply:
undefined will be true with `==`

Reply:
Yes, but the cases API returns an empty email (`""`) if the user has not provided an email.

Reply:
not yet, in @XavierM PR

Reply:
he will resolve